modMcf/src/org/aion/mcf/types/AbstractBlockHeader.java

```diff
@@ -166,6 +170,10 @@ public void setTransactionsRoot(byte[] stateRoot) {
     }
 
     public BigInteger getDifficultyBI() {
+        if (difficulty == null || difficulty.length > DIFFICULTY_LENGTH) {
+            AionLoggerFactory.getLogger("CONS").error("Invalid difficulty length!");
+            return BigInteger.ZERO;
```

Comment:
Can we add a JavaDoc to this that explains that BigInteger.ZERO is used if difficulty length is invalid?  And also mention which range of numbers are valid (I'm guessing it's supposed to be the positive integers, but would be good to make it very clear)